pkg/cmd/experimental/syncgroups/cli/syncgroups.go

```diff
@@ -220,6 +234,29 @@ func (o *SyncGroupsOptions) Complete(typeArg, whitelistFile, blacklistFile, conf
 	return nil
 }
 
+// openshiftGroupNamesOnlyBlacklist returns back a list that contains only the names of the groups.
+// Since Group.Name cannot contain '/', the split is safe.  Any resource ref that is not a group
+// is skipped.
+func openshiftGroupNamesOnlyList(list []string) ([]string, error) {
+	ret := []string{}
+	for _, curr := range list {
+		tokens := strings.SplitN(curr, "/", 2)
+		if len(tokens) == 1 {
+			ret = append(ret, tokens[0])
+			continue
+		}
+
+		// skip invalid tokens
```

Comment:
it's not really skipping the invalid tokens here, should it?  If not, should it gather the errors for the entire list rather than an immediate return?